services/uploads.js

```diff
@@ -113,19 +113,29 @@ Uploads.createUploadCredentials = function() {
  */
 Uploads.createUpload = function(config) {
   v.assertShape({
-    mapId: v.required(v.string),
+    tileset: v.required(v.string),
     url: v.required(v.string),
-    tilesetName: v.string
```

Comment:
if this can also be handled in a backwards compatible way to prevent a breaking change